package.json

```diff
@@ -17,10 +17,10 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "export SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
```

Comment:
Why changes are here in start?

Reply:
To address `error:03000086:digital envelope routines::initialization` error while running App locally.

Reply:
For more info: https://appdividend.com/2022/12/23/error-03000086-digital-envelope-routines-initialization/